.travis/setup_repoman.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Set up directory permissions/portage user and group.
+sudo chmod a+rwX /etc/passwd /etc/group /etc /usr
+echo "portage:x:250:250:portage:/var/tmp/portage:/bin/false" >> /etc/passwd
+echo "portage::250:portage,travis" >> /etc/group
+
+# Shuffle portage repositories around into the right places.
+mkdir -p /etc/portage/repos.conf /usr/coreos-overlay
+mv * /usr/coreos-overlay/
+mv .git /usr/coreos-overlay/
+git clone https://github.com/coreos/portage-stable /usr/portage/
+cp .travis/coreos.conf /etc/portage/repos.conf/
+ln -s /usr/coreos-overlay/profiles/coreos/amd64/sdk /etc/portage/make.profile
+mkdir -p /usr/portage/metadata/{dtd,xml-schema}
+wget -O /usr/portage/metadata/dtd/metadata.dtd https://www.gentoo.org/dtd/metadata.dtd
+wget -O /usr/portage/metadata/xml-schema/metadata.xsd https://www.gentoo.org/xml-schema/metadata.xsd
+
+# Download portage.
+mkdir /tmp/portage && cd /tmp/portage
+wget -qO - "https://github.com/gentoo/portage/archive/portage-${PORTAGE_VER}.tar.gz" | tar xz
```

Comment:
Does this still work now that Gentoo's GitHub project is down?  Maybe use e.g. https://gitweb.gentoo.org/proj/portage.git/snapshot/portage-2.3.41.tar.gz to pull from gentoo.org's Git repo.

Reply:
Indeed it doesn't, using gentoo.org is likely a better solution in general anyway.